app/src/main/java/com/codingblocks/cbonlineapp/util/Certificate.kt

```diff
@@ -0,0 +1,26 @@
+package com.codingblocks.cbonlineapp.util
+
+import android.app.DownloadManager
+import android.content.*
+import android.net.Uri
+import android.os.Environment
+import android.widget.Toast
+
+object Certificate  {
+
+    fun downloadCertificateAndShow(context: Context, certificateUrl: String, fileName: String) {
```

Comment:
I guess we don't an object for this instead make this a function inside viewmodel

Reply:
Sure